code/src/reducers/todos.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+import uniqid from 'uniqid'
+
+const initialState = {
+  items: []
+}
+
+const todos = createSlice({
+  name: 'todos',
+  initialState,
+  reducers: {
+    addToDo: (state, action) => {
+      state.items.unshift({
```

Comment:
This `unshift` method returns an array as reverse?

Reply:
no, it just adds a new element to the beginning of an array. (push adds new elements to the end)